esp32s3-hal/examples/ledc.rs

```diff
@@ -51,15 +51,17 @@ fn main() -> ! {
 
     ledc.set_global_slow_clock(LSGlobalClkSource::APBClk);
 
-    let mut lstimer0 = ledc.get_timer::<LowSpeed>(timer::Number::Timer2);
+    let mut lstimer0 = ledc.get_timer::<LowSpeed>(timer::Number::Timer0);
 
-    lstimer0
-        .configure(timer::config::Config {
-            duty: timer::config::Duty::Duty5Bit,
-            clock_source: timer::LSClockSource::APBClk,
-            frequency: 24u32.kHz(),
-        })
-        .unwrap();
+    let res = lstimer0.configure(timer::config::Config {
+        duty: timer::config::Duty::Duty5Bit,
+        clock_source: timer::LSClockSource::APBClk,
+        frequency: 24u32.kHz(),
+    });
+
+    if res.is_err() {
```

Comment:
Let's clean this up a bit, probably okay to unwrap these errors now?

Reply:
Oh yes ... 🤦‍♂️ ... was just meant for my own testing. Good catch!